pkg/envoy/agent.go

```diff
@@ -191,8 +190,6 @@ func (a *Agent) terminate() {
 						break graceful_loop
 					}
 					log.Warnf("Retrying (%d attempt) to obtain active connections...", retryCount)
-					time.Sleep(retryBackoffTime)
-					retryBackoffTime *= 2 // Exponentially increase the retry backoff time.
```

Comment:
@howardjohn  how about removing sleep with exponential backoff like this? It's probably safer than before.